docs: Fix typo and add contribute section (#9)
apache/paimon-rust: README.md

# Apache Paimon Rust &emsp; [![Build Status]][actions] [![Latest Version]][crates.io]

[Build Status]: https://img.shields.io/github/actions/workflow/status/apache/paimon-rust/ci.yml

> [actions]: https://github.com/apache/paimon-rust/actions?query=branch%3Amain

Reviewer: ```suggestion
[Actions]: https://github.com/apache/paimon-rust/actions?query=branch%3Amain
```

Author: Hi, this label served as markdown link and not readable by users. If we do want to change this, we should change the reference too.

Author: I will ignore this suggestion. Feel free to reopen it if you feel like this change does needed.

[Latest Version]: https://img.shields.io/crates/v/paimon.svg
[crates.io]: https://crates.io/crates/paimon

The rust implementation of Apache Paimon. 

> ## Progress

Reviewer: ```suggestion
## Issue Tracker
```

> See [Tracking issues of paimon rust 0.1.0 release](https://github.com/apache/paimon-rust/issues/3)

Reviewer: ```suggestion
See [Tracking issues of 0.1.0 version for Paimon Rust](https://github.com/apache/paimon-rust/issues/3)
```

## Contributing

> Apache Paimon Rust is under active development. We are always open to people who want to use it or contribute to it. Here are some ways to go.

Reviewer: ```suggestion
Apache Paimon Rust is an exciting project currently under active development. Whether you're looking to use it in your projects or contribute to its growth, there are several ways you can get involved:
```

> - Start with [Contributing Guide](CONTRIBUTING.md).

Reviewer: ```suggestion
- Follow the [Contributing Guide](CONTRIBUTING.md) to contribute.
```

> - Submit [Issues](https://github.com/apache/paimon-rust/issues/new) for bug report or feature requests.

Reviewer: ```suggestion
- Create new [Issue](https://github.com/apache/paimon-rust/issues/new) for bug reportor or feature request.
```

> - Discuss at [dev mailing list](mailto:dev@paimon.apache.org) ([subscribe](<mailto:dev-subscribe@paimon.apache.org?subject=(send%20this%20email%20to%20subscribe)>) / [unsubscribe](<mailto:dev-unsubscribe@paimon.apache.org?subject=(send%20this%20email%20to%20unsubscribe)>) / [archives](https://lists.apache.org/list.html?dev@paimon.apache.org))

Reviewer: ```suggestion
- Start discussion thread at [dev mailing list](mailto:dev@paimon.apache.org) ([subscribe](<mailto:dev-subscribe@paimon.apache.org?subject=(send%20this%20email%20to%20subscribe)>) / [unsubscribe](<mailto:dev-unsubscribe@paimon.apache.org?subject=(send%20this%20email%20to%20unsubscribe)>) / [archives](https://lists.apache.org/list.html?dev@paimon.apache.org))
```
